packages/catalog-process/src/services/validators.ts

```diff
@@ -32,6 +33,36 @@ import {
 } from "../model/domain/errors.js";
 import { ReadModelService } from "./readModelService.js";
 
+export const descriptorStatesAllowingDocumentOperations: DescriptorState[] = [
+  descriptorState.draft,
+  descriptorState.deprecated,
+  descriptorState.published,
+  descriptorState.suspended,
+];
+
+export const updatableDescriptorStates: DescriptorState[] = [
+  descriptorState.deprecated,
+  descriptorState.published,
+  descriptorState.suspended,
+];
+
+export const notActiveDescriptorState: DescriptorState[] = [
+  descriptorState.draft,
+  delegationState.waitingForApproval,
+];
```

Comment:
There are two points in the `readModelService.ts` of the catalog-process where a check as follows is performed - could you reuse this array there?

```
                      $in: [
                        descriptorState.draft,
                        descriptorState.waitingForApproval,
                      ],
```